### PR TITLE
blog: Kaoto 2.3 release

### PR DESCRIPTION
**Pull request by lordrip**

### CODE REVIEW

File: content/blog/2024/12/kaoto-release-2.3.0/index.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+date: 2024-12-10
+draft: false
+authors: [igarashitm, lordrip]
+categories: ["Releases", "Tooling", "Kaoto"]
+preview: "Kaoto 2.3 has been released"
+---
+
+![Kaoto](cover.png)
+
+Following the Apache Camel 4.8.1 release, we're happy to announce the release of Kaoto 2.3.
```
Comment (apupier) [inline content/blog/2024/12/kaoto-release-2.3.0/index.md:12]: 4.8.2? 4.9.0? or maybe better to remove this mention?

**Comment by lordrip** [comment]
Please not merge yet :pray:

**Comment by apupier** [comment]
current errors:
```
[check:html     ] /home/runner/work/camel-website/camel-website/public/blog/2024/12/kaoto-release-2.3.0/index.html
[check:html     ]   1:16144  error  For links within camel.apache.org use relative links, found: https://camel.apache.org/camel-core/getting-started/index.html               camel/relative-links
[check:html     ]   1:17236  error  For links within camel.apache.org use relative links, found: https://camel.apache.org/components/4.8.x/others/langchain4j-tokenizer.html  camel/relative-links

```

File: content/blog/2024/12/kaoto-release-2.3.0/index.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+![Kaoto](cover.png)
+
+Following the Apache Camel 4.9.0 release, we're happy to announce the release of Kaoto 2.3.
+
+## What’s New in Kaoto 2.3?
+
+We're thrilled to announce the release of Kaoto 2.3, bringing new features, improvements, and bug fixes to enhance your integration experience.
+This release also brings the first technical preview of a long awaited feature: the Kaoto DataMapper with the ability to perform data transformations using Camel.
+
+## Here are the key highlights of this release:
```
Comment (apupier) [inline content/blog/2024/12/kaoto-release-2.3.0/index.md:19]: as it is a title, I think I would turn it more in something like:
* Key highlights
* Key highlights of Kaoto 2.3

but on the other had, it seemsvery redundant with the previous "What's new in kaoto 2.3?"

File: content/blog/2024/12/kaoto-release-2.3.0/index.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+In this release, we are introducing the new **Kaoto DataMapper**, a graphical way of authoring data mappings inside your routes using Kaoto.
+
+#### How to use it?
+When creating a Camel route, like any other step, just search for **Kaoto DataMapper** in the catalog and add it from there.
+
+| Tile view                          | List view                          |
+|------------------------------------|------------------------------------|
+| ![Tile](catalogDatamapperTile.png) | ![List](catalogDatamapperList.png) |
+
+Once the **Kaoto DataMapper** step is added in the route, you can launch it using by clicking the `Configure` button in config form.
```
Comment (apupier) [inline content/blog/2024/12/kaoto-release-2.3.0/index.md:31]:
```suggestion
Once the **Kaoto DataMapper** step is added in the route, you can open the DataMapper UI by clicking the `Configure` button in config form.
```

File: content/blog/2024/12/kaoto-release-2.3.0/index.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+#### What's next for the Kaoto DataMapper?
+
+We have a plan to iteratively add more features to the Kaoto DataMapper, for example:
+ - JSON support
+ - Improve XPath expression editor
+   - syntax assist & highlight
+   - better Drag and Drop capability for fields and functions
+ - Data Preview - feed an example input, and instantly see the mapping outcome directly in the UI
+ - Support Setting Camel Variables/Message Headers directly in the UI
+ - Support assuming data structure (schema) by attaching a data example, such as XML instance and JSON instance
```
[the remark this answers is not in the document]
Comment (lordrip) [reply content/blog/2024/12/kaoto-release-2.3.0/index.md:58]: I think it should probably "inferring the data structure by attaching a data example"

**Comment by lordrip** [comment]
Post preview https://pr-1267--camel.netlify.app/blog/2024/12/kaoto-release-2.3.0/

**Review by oscerd — APPROVED** [review]

**Pull request merged**